code/modules/hydroponics/biogenerator.dm

```diff
@@ -16,13 +16,17 @@
 	var/productivity = 0
 	var/max_items = 40
 	var/datum/techweb/stored_research
-	var/list/show_categories = list("Food", "Botany Chemicals", "Organic Materials")
+	var/bio_gen_part_tier
+	var/list/show_categories = list("Food", "Botany Chemicals", "Organic Materials", "Clothing")
```

Comment:
You don't need `var/bio_gen_part_tier` here, just change stored_research to this:
```
var/datum/techweb/specialized/autounlocking/biogenerator/stored_research
```
This will let you access bio_gen_part_tier by just doing `stored_research.bio_gen_part_tier`

Reply:
Removed that variable. Forgot I left that there when I was trying different methods of getting it to work.